extmod/asyncio/stream.py

```diff
@@ -13,11 +14,19 @@ def __init__(self, s, e={}):
     def get_extra_info(self, v):
         return self.e[v]
 
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(self, exc_type, exc, tb):
+        await self.close()
```

Comment:
Close is not async, so how does this work?

Reply:
It doesn't … another merge oversight I'm afraid.